frontend/__tests__/components/route-pages.spec.tsx

```diff
@@ -242,69 +242,3 @@ describe(RouteStatus.displayName, () => {
     expect(wrapper.text()).toEqual('Pending');
   });
 });
-
-describe(RouteWarnings.displayName, () => {
```

Comment:
Replacing these tests with other tests? Or a straight elimination. (I've personally added no tests yet, def want to fix that)

Reply:
We removed/replaced these calculated/parsed warnings with the Conditions table which just shows the raw data/warnings.